db/migrate/20170512162050_create_terms_and_conditions.rb

```diff
@@ -0,0 +1,18 @@
+class CreateTermsAndConditions < ActiveRecord::Migration
+  def change
+    create_table :terms_and_conditions do |t|
+      t.string :name
+      t.string :title
+      t.text :text
+      t.string :version
+      t.boolean :latest, default: false
```

Comment:
I think we would want an index on :name and :scope and maybe just :name by itself. Is the boolean for latest really necessary if you already have version? Won't the highest version automatically be the latest or are you treating this flag as a "counter cache" of sorts.